samcli/local/services/base_local_service.py

```diff
@@ -103,7 +105,9 @@ def get_lambda_output(stdout_stream: io.StringIO) -> Tuple[str, bool]:
         bool
             If the response is an error/exception from the container
         """
-        lambda_response = stdout_stream.getvalue()
+        lambda_response: Union[str, bytes] = stdout_stream_str.getvalue()
+        if stdout_stream_bytes and not lambda_response:
```

Comment:
Is `stdout_stream_bytes` the raw value of `lambda_response` (when `lambda_response` is available)?

Reply:
Yea. Currently we write the response of lambdas to `stdout_stream_str` then it gets read and returned from here, and if the response isn't json deserializeable then we just decode the raw response to utf-8 put it in the `stdout_stream_str` instead. I had to add a second stream (`stdout_stream_bytes`) since the StringIO object can't accept bytes and images can't be decoded to utf-8 without first changing them to base64, which isn't what lambda does (and isn't how SAM previously handled it).